[2021-07-18 04:55:20,247] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2021-07-17T03:39:23.918881+00:00 [queued]>
[2021-07-18 04:55:20,635] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2021-07-17T03:39:23.918881+00:00 [queued]>
[2021-07-18 04:55:20,636] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-18 04:55:20,636] {taskinstance.py:1088} INFO - Starting attempt 4 of 4
[2021-07-18 04:55:20,636] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-18 04:55:20,767] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_1> on 2021-07-17T03:39:23.918881+00:00
[2021-07-18 04:55:20,770] {standard_task_runner.py:52} INFO - Started process 1585 to run task
[2021-07-18 04:55:20,775] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_1', '2021-07-17T03:39:23.918881+00:00', '--job-id', '36', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp4ouznxat', '--error-file', '/tmp/tmpws4f92kh']
[2021-07-18 04:55:20,776] {standard_task_runner.py:77} INFO - Job 36: Subtask task_1
[2021-07-18 04:55:21,971] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_1 2021-07-17T03:39:23.918881+00:00 [running]> on host apache-airflow.us-central1-a.c.eminent-it-info.internal
[2021-07-18 04:55:23,517] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2021-07-17T03:39:23.918881+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-17T03:39:23.918881+00:00
[2021-07-18 04:55:23,518] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-07-18 04:55:23,518] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-07-18 04:55:23,524] {subprocess.py:75} INFO - Output:
[2021-07-18 04:55:26,527] {subprocess.py:83} INFO - Command exited with return code 0
[2021-07-18 04:55:27,407] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_1, execution_date=20210717T033923, start_date=20210718T045520, end_date=20210718T045527
[2021-07-18 04:55:28,113] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-07-18 04:55:28,138] {local_task_job.py:151} INFO - Task exited with return code 0
